src/System.Threading.Timer/tests/System.Threading.Timer.Tests.csproj

```diff
@@ -7,7 +7,7 @@
     <OutputType>Library</OutputType>
     <AssemblyName>System.Threading.Timer.Tests</AssemblyName>
     <ProjectGuid>{ac20a28f-fda8-45e8-8728-058ead16e44c}</ProjectGuid>
-    <NugetTargetMoniker>.NETStandard,Version=v1.3</NugetTargetMoniker>
+    <NugetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.7</NugetTargetMoniker>
```

Comment:
Same here, should we make a change in test.builds file to add a configuration that passes in TargetGroup to be netstandard1.3 and TestTFMs netcoreapp1.0